src/components/Editor/HighlightLine.js

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+// @flow
+import { Component } from "react";
+import { toEditorLine } from "../../utils/editor";
+import { getDocument } from "../../utils/editor/source-documents";
+import { isEqual } from "lodash";
+
+import { connect } from "react-redux";
+import { getVisibleSelectedFrame, getSelectedLocation } from "../../selectors";
+
+import type { Frame, Location } from "debugger-html";
+
+type Props = {
+  editor: Object,
+  selectedFrame: Frame,
+  selectedLocation: Location
+};
+
+export class DebugLine extends Component<Props> {
+  componentDidMount() {
+    const { selectedLocation, selectedFrame, editor } = this.props;
+    // this.setHighlightLine(selectedLocation, selectedFrame, editor);
+  }
```

Comment:
we should be able to remove this function because we dont need to show the highlight when we mount